grails-app/controllers/org/broadinstitute/orsp/AuthenticatedController.groovy

```diff
@@ -347,12 +347,24 @@ class AuthenticatedController implements Interceptor, UserInfo {
         groupedSubmissions
     }
 
-
+    @Deprecated
+    /**
+     * Use getSessionUser().isAdmin instead
+     */
     def isCurrentUserAdmin() {
         render([isAdmin: isAdmin()] as JSON)
     }
 
-    User getSessionUser() {
-        render([getUser()] as JSON)
+    def getSessionUser() {
+        User user = getUser()
+        render([
+            'displayName': user.displayName,
+            'emailAddress': user.emailAddress,
+            'userName': user.userName,
+            'isAdmin': isAdmin(),
+            'isORSP': isORSP(),
+            'isComplianceOffice': isComplianceOffice(),
+        ] as JSON)
```

Comment:
The big changes here are that we return a single object instead of an array and dump in some additional, relevant properties.

Side note: I first tried adding transients to `User`, but `JSON` ignores transient properties. This turned out to be simpler since I can customize the exact properties to return. We should probably not return the db id, role ids, etc.

Reply:
I like this way better than how it was done before. And its reutilizable. :+1:

Reply:
Nice. I'll update our PR with this approach. Thanks!